cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -401,6 +403,55 @@ public synchronized void updateProperties(NamespaceId namespaceId, NamespaceMeta
       throw new BadRequestException(String.format("Mappings %s for namespace %s cannot be updated once the namespace " +
                                                     "is created.", difference, namespaceId));
     }
+
+    if (namespaceMeta.getRepoConfig() != null) {
+      updateRepoConfig(builder, namespaceMeta.getRepoConfig());
+    }
+    
+    NamespaceMeta updatedMeta = builder.build();
+    nsStore.update(updatedMeta);
+    // refresh the cache with new meta
+    namespaceMetaCache.refresh(namespaceId);
+    LOG.info("Namespace {} updated with meta {}", namespaceId, updatedMeta);
```

Comment:
Be careful about logging the `updatedMeta` since it can contain credentials / access token. I would suggest not to log the meta data.

Reply:
The access token is actually not passed in the repository config, it's calling the secure storage API to save from UI

Reply:
Removed the logging of `updatedMeta`